Modules/System Tests/URI/src/UriBuilderTest.Codeunit.al

```diff
@@ -22,6 +22,83 @@ codeunit 135071 "Uri Builder Test"
         Assert.ExpectedError('A call to System.UriBuilder failed with this message: Invalid URI');
     end;
 
+    [Test]
+    [Scope('OnPrem')]
+    procedure SetAndGetSchemeTest()
```

Comment:
Add a test for setting an incorrect scheme. See [here](https://docs.microsoft.com/en-us/dotnet/api/system.uribuilder.scheme?view=net-5.0#System_UriBuilder_Scheme).

You may even introduce an enum to wrap the "allowed" scheme values.